src/index.html

```diff
@@ -11,7 +11,155 @@
+            autocomplete="off" 
+            required
+          >
+        </label>
+        <label class="inputs">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12"
+          >
+        </label>
+        <label class="inputs">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="dateOfBirth" 
+            required
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="terms" 
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="groups">
+        <legend>Registration</legend>
+        <label class="inputs">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            required
+            autocomplete="off"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password:
+          <input 
+            type="password"
+            name="password" 
+            required
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="groups">
+        <legend>An interesting fact about you!</legend>
+        <div class="inputs">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="loveCats" 
+              value="y"
+            >
+            Yes
+          </label>
+          <label>
+            <input 
+              type="radio"
+              name="loveCats" 
+              value="n"
+            >
+             No
+          </label>
+        </div>
+        <label class="inputs">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="favoriteColor"
+          >
+        </label>
+        <label class="inputs">
+          What time do you go to the bed?
+          <input type="time" name="timeToBed">
+        </label>
+        <label class="inputs">
+          What are your favorite brend of cars?
+          <select 
+            name="brendOfCars" 
+            id="brendOfCars" 
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label>
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="rate"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="groups">
+        <legend>Additional info:</legend>
+        <label class="inputs">
+          Comments:
+          <textarea 
+            name="comments" 
+            cols="20" 
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to avoid empty space when click on textarea